packages/design-system-docs/src/pages/components/ChoiceList/Choice.example.jsx

```diff
@@ -1,155 +1,62 @@
-      </Choice>
-      <Choice
-        defaultChecked
-        name="radio_choice_children"
-        type="radio"
-        value="d"
-        checkedChildren={<div className="ds-c-choice__checkedChild">{childDropdown}</div>}
-      >
-        Radio B - with children
-      </Choice>
-    </fieldset>
-
-    <hr className="ds-u-margin-top--2" />
-
-    <fieldset className="ds-c-fieldset">
-      <legend className="ds-c-label">
-        <span>Small checkboxes with children</span>
-      </legend>
-      <Choice name="checkbox_choice_children_small" size="small" type="checkbox" value="a">
-        Checkbox A
-      </Choice>
-      <Choice
-        defaultChecked
-        name="checkbox_choice_children_small"
-        type="checkbox"
-        value="b"
-        checkedChildren={
-          <div className="ds-c-choice__checkedChild ds-c-choice__checkedChild--small">
-            {childDropdown}
-          </div>
-        }
-        size="small"
-      >
-        Checkbox B - with children
-      </Choice>
-      <Choice name="checkbox_choice_children_small" size="small" type="checkbox" value="c">
-        Checkbox C
-      </Choice>
-    </fieldset>
-
-    <div className="ds-base ds-base--inverse ds-u-padding--2 ds-u-margin-top--2">
-      <fieldset className="ds-c-fieldset ds-u-margin-top--0">
-        <legend className="ds-c-label">
-          <span>Inverse radio buttons with children</span>
-        </legend>
-        <Choice name="radio_choice_children_inv" type="radio" value="c">
-          Radio A
-        </Choice>
-        <Choice
-          defaultChecked
-          name="radio_choice_children_inv"
-          type="radio"
-          value="d"
-          checkedChildren={
-            <div className="ds-c-choice__checkedChild ds-c-choice__checkedChild--inverse">
-              {childDropdown}
-            </div>
-          }
-        >
-          Radio B - with children
-        </Choice>
-      </fieldset>
-    </div>
-  </div>,
+  <>
+    <Choice name="checkbox_choice" type="checkbox" label="Checkbox A" value="a" />
+    <Choice
+      defaultChecked
+      name="checkbox_choice_checked"
+      type="checkbox"
+      label="Checkbox B with checked children"
+      value="b"
+      checkedChildren={<div className="ds-c-choice__checkedChild">{childField}</div>}
+    />
+    <Choice
+      defaultChecked
+      name="checkbox_choice_unchecked"
+      type="checkbox"
+      label="Checkbox C with unchecked children"
+      value="c"
+      uncheckedChildren={<div className="ds-c-choice__checkedChild">{childField}</div>}
+    />
+    <Choice
+      name="checkbox_choice_disabled"
+      type="checkbox"
+      label="Disabled Checkbox d"
+      value="d"
+      disabled
+    />
+    <Choice
+      name="checkbox_choice_small"
+      size="small"
+      type="checkbox"
+      label="Small Checkbox E"
+      value="e"
+    />
+    <Choice name="radio_choice" type="radio" label="Radio A" value="a" />
+    <Choice
+      name="radio_choice_checked"
```

Comment:
Use the same `name=` for radio buttons here so they can be unchecked 

```suggestion
      name="radio_choices"
```